docker-compose-e2e-tests.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.7"
+
+name: mit-open-e2e-tests
+
+services:
+  db:
+    image: postgres:12.17
+    container_name: mit-open-e2e-test-db
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_PASSWORD=postgres
+    volumes:
+      - pgdata-e2e-tests:/var/lib/postgresql/data
+
+  nginx:
+    image: nginx:1.25.3
+    ports:
+      - 8063:8063
+    links:
+      - web
+    volumes:
+      - ./config/nginx.conf:/etc/nginx/conf.d/web.conf
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8063"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+
+  web:
+    build:
+      context: .
+    environment:
+      PORT: 8061
+    env_file: .env.ci
+    command: ./scripts/run-django-dev.sh
```

Comment:
I think it might be possible to use a different database, recreate it on each run, and then populate it without relying on a nested docker container. 

Modify `.env.ci:
```
DATABASE_URL=postgres://postgres:postgres@db:5432/e2e_postgres. # different db name
PGPASSWORD=postgres
```

Then adjust the docker-compose-e2e-tests.yml file, adding a health check to postgres to ensure it's accepting connections, then running commands from the web container to recreate the e2e test database:

```yaml
version: "3.7"

services:
  db:
    image: postgres:12.17
    ports:
      - 5432:5432
    environment:
      - POSTGRES_PASSWORD=postgres
    healthcheck:
      test: ["CMD", "pg_isready", "-U", "postgres"]
      interval: 5s
      timeout: 5s
      retries: 2

  nginx:
    image: nginx:1.25.3
    ports:
      - 8063:8063
    links:
      - web
    volumes:
      - ./config/nginx.conf:/etc/nginx/conf.d/web.conf
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:8063"]
      interval: 5s
      timeout: 5s
      retries: 5

  web:
    build:
      context: .
    environment:
      PORT: 8061
    env_file: .env.ci
    command: >
      bash -c "dropdb --if-exists -h db -U postgres e2e_postgres && createdb -h db -U postgres e2e_postgres && ./scripts/run-django-dev.sh"
    stdin_open: true
    tty: true
    ports:
      - 8061:8061
    links:
      - db
    depends_on:
      db:
        condition: service_healthy

  e2e-tests:
    build:
      context: e2e_testing
    environment:
      - CI=true
      - BASE_URL=http://nginx:8063
    depends_on:
      nginx:
        condition: service_healthy
    links:
      - nginx
```